app/src/main/java/com/example/reduxtestapp/ui/country/CountryScreen.kt

```diff
@@ -189,6 +189,10 @@ private fun SearchBar(
         onSendSearchQuery(searchQuery)
     }
 
+    LaunchedEffect(Unit) {
+        onSendSearchQuery("")
+    }
```

Comment:
if this triggers a network call (and potentially other stuff, like analytics) then it's not a good way to clear the search text field. think of some alternatives